web/src/components/ModalTwoButton.tsx

```diff
@@ -18,31 +20,37 @@ interface Props {
 }
 
 export const ModalTwoButton = (props: Props): ReactElement => {
+  const isMobileAndUp = useMediaQuery(MediaQueries.mobileAndUp);
+
   const buttonNode = (
     <div
-      className="padding-x-4 padding-y-3 bg-base-lightest display-flex flex-column flex-justify-center mobile-lg:flex-row"
+      className={`padding-x-4 padding-y-3 bg-base-lightest ${
+        isMobileAndUp ? "flex flex-column flex-align-end" : ""
```

Comment:
I feel like this would be better as a mixin that way we don't have to do JS media queries which I think are kinda an anti pattern

Reply:
I created this chore, let's talk about it during the next grooming